### PR TITLE
fzf 0.9.0

### PR DESCRIPTION
**Pull request by junegunn**

### CODE REVIEW

File: Library/Formula/fzf.rb [hunk 1]
```diff
@@ -2,13 +2,16 @@
 
 class Fzf < Formula
   homepage "https://github.com/junegunn/fzf"
-  url "https://github.com/junegunn/fzf/archive/0.8.9.tar.gz"
-  sha1 "efbe5955aa09e9d94f304c6177e55060b1926e28"
+  url "https://github.com/junegunn/fzf-bin/releases/download/0.9.0/fzf-0.9.0-homebrew.tgz"
```
Comment (mikemcquaid) [inline Library/Formula/fzf.rb]: This should build from source instead.

Comment (junegunn) [inline Library/Formula/fzf.rb]: @mikemcquaid Then it requires Go 1.4. Do you think it's preferable even for the users who don't ever need Go for themselves?

Comment (mikemcquaid) [inline Library/Formula/fzf.rb]: Yeh, that's fine. We have other formulae with Go as a build-time dep.

Comment (junegunn) [inline Library/Formula/fzf.rb]: Updated the formula. It seems to work but I'm not sure if I'm doing the right way. Please let me know if there's something more to be done.

**Comment by mikemcquaid** [comment]
Looks good to me and the test is passing. Thanks!

**Comment by junegunn** [comment]
Thanks for the review! Is there anything else I have to do?

**Pull request closed without merging**